### PR TITLE
Did a change to refresh page every 2 minute.

### PR DESCRIPTION
**Pull request by rishabhmaheshwari7**
This feature I thought would be great as sometimes when I come back to this page 
I had to manually refresh the page to see my progress.

### CODE REVIEW

**Comment by subodhk01** [comment]
Hey @rishabhmaheshwari7, great change. Just one thing, don't you think refresh should be added to the ladder page(ladder.html) rather than on homepage (index.html)?

**Comment by rishabhmaheshwari7** [comment]
Oh yes...sorry about that..
I'll make the changes right now

File: templates/index.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
         <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
         <link rel="stylesheet" href="{% static 'css/index.css' %}">
+        <meta http-equiv="refresh" content="120" >
```
Comment (subodhk01) [inline templates/index.html:14]: You can remove this meta tag coz this one is not required

**Review by subodhk01 — CHANGES_REQUESTED** [review]

File: templates/index.html (hunk 1 above)
Comment (rishabhmaheshwari7) [reply templates/index.html]: Did the changes you asked for.
Thanks for pointing out the mistake

**Comment by subodhk01** [comment]
Bro you made an unnecessary change, it would be helpful you can remove all the changes in index.html (So that it shows only 1 file changed) as I have to revert a previous commit in that file so do not want any history modifications

**Comment by subodhk01** [comment]
Bro I think you deleted the file 😅

**Comment by rishabhmaheshwari7** [comment]
As I am new to this so I got a bit confused
Thus made a new pull request
You can ignore this pull request
Sorry for the inconvenience

Thank you😄

**Comment by rishabhmaheshwari7** [comment]
Please check the other pull request I made.
Thanks

**Pull request closed without merging**